compiler/rustc_codegen_ssa/src/base.rs

```diff
@@ -15,8 +15,9 @@ use rustc_attr as attr;
 use rustc_data_structures::fx::FxHashMap;
 use rustc_data_structures::profiling::{get_resident_set_size, print_time_passes_entry};
 
+use rustc_data_structures::sync::par_iter;
```

Comment:
where is `par_iter` called in `cfg(not(parallel_compiler))`?

Reply:
I think it moved because in other case in will fail to compile, the reason is `cfg!(parallel_compiler)`. If return back `#[cfg(parallel_compiler)]` all should work (at least i remembering something like that when i touched that part of code).

Reply:
My bad, I didn't see the `par_iter` call in the hidden section between the diffs.

Reply:
I switched from `#[cfg]` to `cfg!()` to ensure that changes to the code are always typechecked and not just when explicitly running in the parallel_compiler configuration.